CONTRIBUTING.md

```diff
@@ -112,6 +112,18 @@ Core developers should follow these rules when processing pull requests:
 * Use "[Squash and merge](https://github.com/blog/2141-squash-your-commits)"
   to merge PRs.
 * Delete branches for merged PRs (by core devs pushing to the main repo).
+* Edit the final commit message before merging to conform to the following
+  style (we wish to have a clean `git log` output):
+  * When merging a multi-commit PR make sure that the commit message doesn't
```

Comment:
I would add something to make sure that if there's an issue being fixed to ensure that "Fixed #xxx" occurs. (And not in the title!)